src/ploomber/cli/ploomber_io.py

```diff
@@ -62,7 +62,7 @@ def wrapper(catch_exception=True, **kwargs):
     return wrapper
 
 
-# FIXME: capture only certain types of exceptions. If it's something we dind't
+# FIXME: capture only certain types of exceptions. If it's something we didn't
```

Comment:
why was this renamed from io -> ploomber_io?

Reply:
this is causing some merge conflicts so if there's isn't a strong reason to rename i'd say not to. but if we want to rename, it should be something that doesn't have "ploomber" as the module name, that's a bit weird

Reply:
This was a pycharm bug if I recall correctly. We can keep it as is to avoid the conflicts

Reply:
what error? renaming is going to cause conflicts as well. there are other branches that have updated this module

Reply:
oh and I just remembered that the `io` module is part of the public API, if we change this, we'll break the code of some users